data/ecosystems/a/avalanche.toml

```diff
@@ -554,8 +555,8 @@ url = "https://github.com/ava-labs/chains"
 url = "https://github.com/ava-labs/chainsafe-scripts"
 
 [[repo]]
-url = "https://github.com/ava-labs/coreth"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/coreth"
```

Comment:
can you keep the tags parameter below the URL like it was originally? not sure why it was changed